Bottom: b3ea573c1045677187bbd59f43813b03d866d196
Top:    b7d7ea308c9bc9debf3ef071c790fce4f1c16d23
Author: Jon Smirl <jonsmirl@smirl2.(none)>
Date:   2013-06-22 00:26:02 -0400

Refresh of asiarf

---

diff --git a/files/etc/config/network b/files/etc/config/network
index 35d4e95..f3bfb81 100644
--- a/files/etc/config/network
+++ b/files/etc/config/network
@@ -5,39 +5,18 @@ config interface 'loopback'
 	option netmask '255.0.0.0'
 
 config globals 'globals'
-	option ula_prefix 'fdd5:b76f:792d::/48'
+	option ula_prefix 'fd28:7772:4f77::/48'
+
+config switch
+	option name 'switch0'
+	option reset '1'
+	option enable_vlan '0'
 
 config interface 'lan'
-	option ifname 'eth0.1'
+	option ifname 'eth0'
 	option type 'bridge'
 	option proto 'static'
-	option ipaddr '192.168.2.200'
+	option ipaddr '192.168.2.2'
 	option netmask '255.255.255.0'
-	option ip6assign '64'
-	option gateway '192.168.2.1'
-	option dns '192.168.2.1'
-
-config interface 'wan'
-	option ifname 'eth0.2'
-	option proto 'dhcp'
-
-config interface 'wan6'
-	option ifname '@wan'
-	option proto 'dhcpv6'
-
-config switch
-	option name 'rt305x'
-	option reset '1'
-	option enable_vlan '1'
-
-config switch_vlan
-	option device 'rt305x'
-	option vlan '1'
-	option ports '0 1 2 3  6t'
-
-config switch_vlan
-	option device 'rt305x'
-	option vlan '2'
-	option ports '4  6t'
-
+	option ip6assign '60'
 
diff --git a/target/linux/ramips/dts/AWM002.dts b/target/linux/ramips/dts/AWM002.dts
deleted file mode 100644
index adadfa1..0000000
--- a/target/linux/ramips/dts/AWM002.dts
+++ /dev/null
@@ -1,108 +0,0 @@
-/dts-v1/;
-
-/include/ "rt5350.dtsi"
-
-/ {
-	compatible = "AWM002", "ralink,rt5350-soc";
-	model = "AsiaRF AWM002";
-
-	palmbus@10000000 {
-		sysc@0 {
-			ralink,pinmux = "spi", "uartlite", "jtag", "mdio", "sdram", "rgmii";
-			ralink,gpiomux = "i2c";
-			ralink,uartmux = "gpio";
-			ralink,wdtmux = <1>;
-		};
-
-		gpio0: gpio@600 {
-			status = "okay";
-		};
-
-		spi@b00 {
-			status = "okay";
-			m25p80@0 {
-				#address-cells = <1>;
-				#size-cells = <1>;
-				compatible = "mx25l6405d";
-				reg = <0 0>;
-				linux,modalias = "m25p80", "mx25l6405d";
-				spi-max-frequency = <10000000>;
-
-		partition@0 {
-			label = "u-boot";
-			reg = <0x0 0x30000>;
-			read-only;
-		};
-
-		partition@30000 {
-			label = "u-boot-env";
-			reg = <0x30000 0x10000>;
-			read-only;
-		};
-
-		factory: partition@40000 {
-			label = "factory";
-			reg = <0x40000 0x10000>;
-			read-only;
-		};
-
-		partition@50000 {
-			label = "firmware";
-			reg = <0x50000 0x1fb0000>;
-		};
-			};
-		};
-	};
-
-	ethernet@10100000 {
-		status = "okay";
-	};
-
-	esw@10110000 {
-		status = "okay";
-		ralink,portmap = <0x3f>;
-	};
-
-	wmac@10180000 {
-		status = "okay";
-	};
-
-	ehci@101c0000 {
-		status = "okay";
-	};
-
-	ohci@101c1000 {
-		status = "okay";
-	};
-
-	i2c-gpio {
-		compatible = "i2c-gpio";
-
-		gpios = <&gpio0 1 0 &gpio0 2 0>;
-		i2c-gpio,delay-us = <10>;
-
-		pcf0: iexp@38 {
-			#gpio-cells = <2>;
-			compatible = "inxp,pcf8574a";
-			reg = <0x38>;
-			gpio-controller;
-		};
-
-		hwmon@4b {
-			compatible = "national,lm92";
-			reg = <0x4b>;
-		};
-	};
-
-	gpio-leds {
-		compatible = "gpio-leds";
-		ld1 {
-			label = "asiarf:green:ld1";
-			gpios = <&pcf0 0 1>;
-		};
-		ld2 {
-			label = "asiarf:green:ld2";
-			gpios = <&pcf0 1 1>;
-		};
-	};
-};
